index.html

```diff
@@ -4820,6 +4820,113 @@ <h3>Verification</h3>
 
       </section>
 
+      <section class="normative">
+        <h3>Context Validation</h3>
+
+        <p>
+This section contains an algorithm that [=verifiers=] MUST run after running the
+algorithm in Section [[[#verification]]], when validating a [=verifiable
+credential=] or a [=verifiable presentation=]. This algorithm takes a document
```

Comment:
See my comment in https://github.com/w3c/vc-data-integrity/pull/291#discussion_r1687883171. Any reason why this algorithm shouldn't be run before even touching verification?

Reply:
It seems to me that it could be up to an implementer whether it is run before or after verification. This an application-level check, and the best time to run it can be application / use-case dependent.

Reply:
Than the text should say so! It currently has a numbered list, which suggests mandatory ordering...

Reply:
Fixed in 8dbad0b15b3b4d2ad46f6ba381951b6d73ca8990.